code/modules/turbolift/turbolift.dm

```diff
@@ -0,0 +1,266 @@
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// 												D O C U M E N T A T I O N  			K m c 2 0 0 0 -> 22/01/2019																				//
+// Turbolifts and you! How to make elevators with little to no effort.																														//
+// ENSURE that the turbolift object ITSELF lines up with the others. This is to prevent the panel jumping about wildly and looking stupid													//
+// If you want to make a multi door turbolift, place the controls at least 1 tile away from the doors. That way it switches to the more CPU intensive area based door acquisition system 	//
+// This is area based! Ensure each turbolift is in a unique area, or things will get fucky																									//
+// Ensure that turbolift doors are at least one tile away from the next lift. See DeepSpace13.dmm for examples. 																			//
+// Use the indestructible elevator turfs or it'll look terrible!				  																											//
+// Modify pixel_x and y as needed, it starts off snapped to the tile below a wall.																											//
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+
+//Areas//
+
+/area/turbolift //Only use subtypes of this area
+	requires_power = FALSE //no APCS in the lifts please
+	ambientsounds = list('sound/effects/turbolift/elevatormusic.ogg')
+
+/area/turbolift/primary
+	name = "primary turbolift"
+
+/area/turbolift/secondary
```

Comment:
Are these really needeed

Reply:
Does each lift need a unique area? Yes. Do these subtypes need to exist? Not really, it's just future-proofing for maps with multiple lifts.

Reply:
Why does each lift need a unique area?